demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1302,21 +1352,44 @@ def test_modeling_rule(
             interactive,
             ctx,
         )
-        if not success:
+        if success:
+            logger.info(
+                f"[green]Test Modeling rule {get_relative_path_to_content(modeling_rule_directory)} passed[/green]",
+                extra={"markup": True},
+            )
+        else:
             errors = True
             logger.error(
-                f"[red]Error testing modeling rule {get_relative_path_to_content(modeling_rule_directory)}[/red]",
+                f"[red]Test Modeling rule {get_relative_path_to_content(modeling_rule_directory)} failed[/red]",
```

Comment:
Either Modeling Rule, or modeling rule. I like the capitalization.
```suggestion
                f"[red]Test Modeling Rule {get_relative_path_to_content(modeling_rule_directory)} failed[/red]",
```